.travis.yml

```diff
@@ -62,6 +62,42 @@ matrix:
             - libboost-log1.55-dev
             - ninja-build
 
+    # Clang 4.0
+    - compiler: clang
+      env: C_COMPILER=clang-4.0 CXX_COMPILER=clang++-4.0
+      addons:
+        apt:
+          sources:
+            - boost-latest
+            - ubuntu-toolchain-r-test
+            - llvm-toolchain-trusty-4.0
```

Comment:
Given that Trusty is the default Travis environment now, could you please also replace the "llvm-toolchain-precise" packages for Trusty ones?